components/desktop/lightdm/lightdm.p5m

```diff
@@ -57,7 +58,8 @@ link path=usr/lib/$(MACH64)/liblightdm-gobject-1.so.0 \
 file path=usr/lib/$(MACH64)/liblightdm-gobject-1.so.0.0.0
 file path=usr/lib/$(MACH64)/lightdm-guest-session mode=0555
 file path=usr/lib/$(MACH64)/pkgconfig/liblightdm-gobject-1.pc
-file path=usr/sbin/lightdm
+# LightDM needs to be updated in new BE, otherwise X crashes
```

Comment:
crashes or is killed?

Reply:
First three warnings is "the usual stuff". I assume killed:

```
** (process:20972): WARNING **: 11:06:21.856: Error getting user list from org.freedesktop.Accounts: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Accounts was not provided by any .service files
** (process:20972): WARNING **: 11:06:21.988: Error getting XDG_RUNTIME_DIR from ConsoleKit: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Method "GetXDGRuntimeDir" with signature "" on interface "org.freedesktop.ConsoleKit.Session" doesn't exist
** (lightdm:20928): WARNING **: 11:06:21.989: Error activating ConsoleKit session: GDBus.Error:org.freedesktop.DBus.GLib.UnmappedError.CkSeatError.Code0: Activation is not supported for this kind of seat
end from FAM server connection
end from FAM server connection
Failed to get D-Bus connection
** (process:20972): WARNING **: 11:06:54.608: Error ending ConsoleKit session: The connection is closed
```

Reply:
please, update comment.

Reply:
Anyway, I'm not sure about reboot-needed for lightdm, as efficiently it is not needed.
What are we trying to achieve? Avoid restarting service on pkg update? But given hipster's nature, pkg update will likely also update kernel module and will create new BE anyway. But if we put reboot-needed on lightdm, it will make developing/testing it a bit harder - you'll have to reboot after each update of lightdm.